tests/artists/test_scatter.py

```diff
@@ -0,0 +1,35 @@
+import numpy as np
+import WrightTools as wt
+import matplotlib.pyplot as plt
+
+
+def test_scatter_broadcast_vars():
+    data = wt.data.Data(name="data")
+    rng1 = np.random.rand(50)[:, None]
+    rng2 = np.random.rand(50)[None, :]
+    data.create_variable("x", values=2 * rng1 - 1)
+    data.create_variable("y", values=2 * rng2 - 1)
+    data.create_channel("z", values=np.exp(-(data.x[:] ** 2 + data.y[:] ** 2)))
+    fig, gs = wt.artists.create_figure()
+    ax = plt.subplot(gs[0])
+    ax.scatter(data, "x", "y", channel="z", s=20, alpha=0.5)
```

Comment:
These calls don't do what you think... if you swapped x and y in the function call, current implementation would not flip your axes, I believe